app/js/components/drugOrderEntry/ActiveOrders.jsx

```diff
@@ -276,10 +284,12 @@ ActiveOrders.defaultProps = {
 };
 
 
-const mapStateToProps = ({ activeOrderReducer }) => ({
+const mapStateToProps = ({ activeOrderReducer, orderSelectionReducer }) => ({
```

Comment:
As opposed to having a new reducer for this feature, is there any chance we can take advantage of an already existing reducer that is closely related to this implementation? Just a suggestion.